csvqb/csvqb/tests/unit/configloaders/infojsontests.py

```diff
@@ -21,33 +23,160 @@ def test_csv_cols_assumed_dimensions(self):
         data = pd.read_csv(self.get_test_cases_dir() / "configloaders" / "data.csv")
         cube = get_cube_from_info_json(self.get_test_cases_dir() / "configloaders" / "info.json", data)
 
-        matching_columns = [c for c in cube.columns if c.csv_column_title == "Undefined Column"]
-        self.assertEqual(1, len(matching_columns))
-        undefined_column_assumed_definition: CsvColumn = matching_columns[0]
+        # matching_columns = [c for c in cube.columns if c.csv_column_title == "Undefined Column"]
+        # self.assertEqual(1, len(matching_columns))
+        # undefined_column_assumed_definition: CsvColumn = matching_columns[0]
+        #
+        # if not isinstance(undefined_column_assumed_definition, QbColumn):
+        #     raise Exception("Incorrect type")
+        #
+        # self.assertIsInstance(undefined_column_assumed_definition.component, NewQbDimension)
+        # new_dimension: NewQbDimension = undefined_column_assumed_definition.component
+        # self.assertIsNotNone(new_dimension.code_list)
+        #
+        # if not isinstance(new_dimension.code_list, NewQbCodeList):
+        #     raise Exception("Incorrect type")
+        #
+        # newly_defined_concepts = list(new_dimension.code_list.concepts)
+        #
+        # self.assertTrue(1, len(newly_defined_concepts))
+        #
+        # new_concept = newly_defined_concepts[0]
+        # self.assertEqual("Undefined Column Value", new_concept.label)
+
+        errors = cube.validate()
+        errors += validate_qb_component_constraints(cube)
+
+        self.assert_no_validation_errors(errors)
+
+    def test_csv_bottles_assumed_dimensions(self):
```

Comment:
Would be good to have a name which better describes what we're testing here. 

Perhaps you could explain what it is that we're testing and we'll think of a name based on that? 

I don't think it should refer to the individual test case you're using. i.e. it shouldn't refer to 'bottles' at all as that's incidental to what we're testing here.

Reply:
renamed to `def test_multiple_measures_and_units_loaded_in_uri_template(self):`